core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerImpl.java

```diff
@@ -269,19 +271,19 @@ public PlayerListenersHolder getListenersHolder() {
     }
 
     @Override
-    public void reset() {
-        release();
+    public void stop() {
+        mediaPlayer.stop();
     }
 
     @Override
-    public void stop() {
-        mediaPlayer.stop();
+    public void reset() {
+        release();
```

Comment:
This is weird...calling `release()` for both the `reset` and the `release`. We need to investigate this when we have finished writing our tests for the `MediaPlayer`.